tests/50federation/33room-get-missing-events.pl

```diff
@@ -377,21 +377,49 @@ sub sytest_user_and_room_fixture {
             Future->done(1);
          }),
       )->then( sub {
-         # wait for S to turn up
-         await_sync_timeline_contains(
-            $creator_user, $room2->room_id, check => sub {
-               my ( $event ) = @_;
-               log_if_fail "Got event in room2", $event;
+         # the server may send a /state request; be prepared to answer that.
+         # (it may, alternatively, send individual /event requests)
+         my $state_req_fut = $inbound_server->await_request_v1_state(
+            $room2->{room_id}, $event_id_Q,
+         )->then( sub {
+            my ( $req, @params ) = @_;
+            log_if_fail "/state request", \@params;
 
-               my $event_id = $event->{event_id};
+            my $resp = {
+               state => [ values( %initial_room2_state ) ],
```

Comment:
Should this key be `pdus` or am I reading [the spec](https://spec.matrix.org/v1.2/server-server-api/#get_matrixfederationv1stateroomid) wrong?

```suggestion
               pdus => [ values( %initial_room2_state ) ],
```

Same for the 3 other places in the PR

Reply:
nope, you're right :-S